src/components/Notifications.tsx

```diff
@@ -0,0 +1,58 @@
+import { useMemo, useState } from "react";
+import useOnclickOutside from "react-cool-onclickoutside";
+
+interface NotificationsProps {
+  notifications: string[];
+  emptyNotificationText: string;
+}
+
+export const Notifications = ({
+  notifications,
+  emptyNotificationText,
+}: NotificationsProps) => {
+  const count = notifications.length;
+  const [openNotification, setOpenNotification] = useState(false);
+  const notificationsRef = useOnclickOutside(() => setOpenNotification(false));
+
+  const NotificationWrapper = useMemo(() => {
+    const handleToggleNotification = () => {
+      setOpenNotification((prev) => !prev);
+    };
+
+    return ({ children }: any) => (
+      <div ref={notificationsRef}>
+        <span
+          className="user-menu--open active"
+          data-count={count}
```

Comment:
Si lo seteas con 0, se va a mostrar el 0 en el icon, creo que lo esperado es mostrar cantidad solo cuando es >1

Reply:
Si tienes razón, conviene setearlo en nulo cuando es menor que 1 el _count_. algo así 👇🏻 
```suggestion
          data-count={count ? count : null}
```